### PR TITLE
Leaderboard App with APIs

### PR DESCRIPTION
**Pull request by sabetech**
In this pull request
- Made an API request to get the game id
- Used game id to save games and get saved game scores
- Displayed existing scores in the list

### CODE REVIEW

File: src/leaderboard.js [hunk 1]
```diff
@@ -1,20 +1,78 @@
...
+            'Content-Type': 'application/json',
+          },
+        });
+        const jsonresponse = await response.json();
+        this.scores = jsonresponse.result;
+        this.render();
+      } catch (e) {
+        throw e.message();
+      }
     }
```
Comment (Danieldotcomcoder) [inline src/leaderboard.js:69]: I hope you're doing well 😀, when the user clicks the refresh button, it should update the scores list  after receiving data from API, in your case it's not working, Kindly Fix it

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @sabetech , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Professional Readme File ✔✔
- Linters are setup correctly ✔✔
- Clear and nice code ✔✔

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/leaderboard.js (hunk 1 above)
Comment (sabetech) [reply src/leaderboard.js:69]: Thanks for the review. 
I have seen the bug. It was a crucial oversight. thanks again for pointing that out.

**Review by EricMbouwe — APPROVED** [review]
### Hi @sabetech, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**